runtime/ftplugin/man.vim

```diff
@@ -1,37 +1,27 @@
-" Vim filetype plugin file
-" Language:	man
-" Maintainer:	SungHyun Nam <goweol@gmail.com>
-
-if has('vim_starting') && &filetype !=# 'man'
-  finish
-endif
-
-" Only do this when not done yet for this buffer
 if exists('b:did_ftplugin')
   finish
 endif
 let b:did_ftplugin = 1
 
-" Ensure Vim is not recursively invoked (man-db does this)
-" when doing ctrl-[ on a man page reference.
-if exists('$MANPAGER')
-  let $MANPAGER = ''
-endif
-
-setlocal iskeyword+=\.,-,(,)
-
-setlocal buftype=nofile noswapfile
-setlocal nomodifiable readonly bufhidden=hide nobuflisted tabstop=8
-
-if !exists("g:no_plugin_maps") && !exists("g:no_man_maps")
-  nnoremap <silent> <buffer> <C-]>    :call man#get_page(v:count, expand('<cword>'))<CR>
-  nnoremap <silent> <buffer> <C-T>    :call man#pop_page()<CR>
+setlocal buftype=nofile
+setlocal noswapfile
+setlocal nofoldenable
+setlocal bufhidden=hide
+setlocal nobuflisted
+setlocal nomodified
+setlocal readonly
+setlocal nomodifiable
+setlocal noexpandtab
+setlocal tabstop=8
+setlocal softtabstop=8
+setlocal shiftwidth=8
+setlocal nolist
+setlocal foldcolumn=0
+setlocal colorcolumn=0
+
+if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
+  nnoremap <silent> <buffer> <C-]>    :call man#get_page(g:man_find_window, 'edit')<CR>
+  nmap     <silent> <buffer> <K>      <C-]>
+  nnoremap <silent> <buffer> <C-t>    :call man#pop_tag()<CR>
   nnoremap <silent> <nowait><buffer>  q <C-W>c
-  if &keywordprg !=# ':Man'
-    nnoremap <silent> <buffer> K      :call man#get_page(v:count, expand('<cword>'))<CR>
```

Comment:
I'll remove it. Is the count feature really worth it, it won't even work for all man page sections like `n` and `0P`. And I'm getting very weird behaviour with the man plugin and using the count. Random errors about it not being able to parse random text as it selects the wrong text underneath the cursor. Or when a error happens, its printed as many times as the count lol. I really don't think we should implement it.